docs/core/idioms/flow-to-flow.md

```diff
@@ -63,9 +64,9 @@ with Flow("parent-flow") as flow:
 ## Scheduling a Flow-of-Flows
 
 Oftentimes different people are responsible for maintaining different flows; in this case it can be useful
-to construct a Flow-of-Flows that specifies stateful dependencies between various Flows.  The `wait` kwarg
-allows you to specify that the task should wait until the triggered flow run completes, and reflect the
-flow run state as the task state.
+to construct a Flow-of-Flows that specifies stateful dependencies between various Flows.  
+The `wait_for_flow_run` task allows you to specify that the task should wait until the triggered flow run completes, 
+and reflect the flow run state as the task state.
```

Comment:
```suggestion
The `wait_for_flow_run` task allows you to specify that the task should wait until the triggered flow run completes.
```

This task does not raise the state of the child flow like `StartFlowRun(wait=True)` did.

Reply:
Does the rest of the example make sense with this caveat?

Reply:
Oh this is interesting!
So, I would I replicate the `StartFlowRun` behaviour using `creat_flow_run` and `wait_for_flow_run` ?

Reply:
It'll wait for the final state and it'll return a `FlowRunView` containing the final state, but you'd need a third task that raises the status (or we could add a bool to `wait_for_flow_run` which may make sense).

Reply:
We'll follow your suggestion regarding `FlowRunView`.
However, I think that having a bool in `wait_for_flow_run` would greatly improve its usability.

Reply:
> Does the rest of the example make sense with this caveat?

Probably not, I'm going to remove it.
I'll also accept the suggested change above.

Reply:
I ended up accepting the suggested change, with a small adjustment that imho makes the rest of example useful.